lua/ide/panels/panel.lua

```diff
@@ -274,6 +276,28 @@ Panel.new = function(tab, position, components)
         for _, f in ipairs(restores) do
             f()
         end
+
+        if not self.has_opened_once then
+            self.has_opened_once = true
+            for _, component in ipairs(self.components) do
+                local default_height = vim.tbl_get(component, 'config', 'default_height')
+                if default_height ~= nil then
+                    if default_height > vim.o.lines then
+                        vim.defer_fn(function()
+                          vim.notify(string.format(
```

Comment:
I think I'm actually wrong about this. I tested it it all out without this check, and it looks like Neovim will bound the the height. 

The issue I was referring to happens when you're splitting windows, one after the other, to make a stack. If you don't equal out all the windows while you're split, you'll recursively make smaller and smaller wins which then gives a "No more room" split error.

Since you are doing this as a one time pass after the windows have been setup, I think this can go away.

Reply:
Oh cool 👍